src/views/Restoration/RestorationBusinessInformation.vue

```diff
@@ -202,6 +108,18 @@ export default class RestorationBusinessInformation extends Vue {
     })
   }
 
+  private isEmptyRecordsAddress () : boolean {
+    if (!this.addresses.recordsOffice?.mailingAddress || !this.addresses.recordsOffice?.deliveryAddress) {
+      return true
+    }
```

Comment:
1. this method doesn't return a value if the statement is false
2. this could be a getter instead of a method
3. should you check the sub-objects?

Reply:
Can make it a get and return if false. Need to check sub-objects as they are undefined.